CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-20)
+--------------------------
+
+- Adds user secrets management mechanism.
+- Adds `--workflow` flag for specifying workflow for `run` and `create`
```

Comment:
For Restructured Text it's [double backquote](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#inline-markup-and-special-characters-e-g-bold-italic-verbatim).